data/tutorials/guides/1wf_00_compiling_ocaml_projects.md

```diff
@@ -36,9 +36,59 @@ system, which provides the `ocamlfind` command.
 
 OCaml comes with two compilers: `ocamlc` is the bytecode compiler, and
 `ocamlopt` is the native code compiler. If you don't know which one to use, use
-`ocamlopt` since it provides executables that are faster than bytecode.
+`ocamlopt` since it provides executables that are faster than bytecode. 
 
-Let's assume that our program `program` has two source files,
+But if you want to try `ocamlc`, you can go ahead and try the following.
+
+Create a new directory named hello-ocaml and navigate to the directory:
+
+```dune
+$ mkdir hello-ocaml
+$ cd hello-ocaml
+```
+Next, create a file named hello.ml and add the following code with your favorite text editor:
+
+```dune
+let _ = print_endline "Hello OCaml!"
+```
+
+**Note:** There are no double semicolons ;; at the end of that line of code.
+
+The *let _* = signifies that we don't care to give a name to the code on the right-hand side of =, hence the _.
+
+Now, we are ready to run the code. Save the file and return to the command line. Let's compile the code:
+
+```dune
+ocamlc -o hello hello.ml
+```
+
+The -o hello option tells the compiler to name the output executable as hello. The executable hello contains compiled OCaml bytecode. In addition, two other files are produced, hello.cmi and hello.cmo. 
+
+`hello.cmi` contains compiled interface information for OCaml modules. An interface file includes type information and module signatures but doesn't contain the actual code. 
+
+`hello.cmo` contains the compiled bytecode for OCaml modules. Bytecode is an intermediate representation of the code that is executed by the OCaml interpreter or runtime system.
+
+`Note:` _cmi_ stands for Compiled Module Interface and _cmo_ stands for Compiled Module Object.
+
+Both of these files play crucial roles in the OCaml build process.
+
+Now let's run the executable and see what happens:
+
+```dune
```

Comment:
````shell
```dune
````

Reply:
I think there is nothing to change here? A space maybe?